spec/usermanual/customcourse.html.erb

```diff
@@ -278,6 +286,14 @@ require 'cgi' # for escapeHTML
   <h2>Personalized course progression</h2>
 
   <p>
+    <i>
+      Notice that course progression can now be configured from the webapp.
+      Configuration from the repository is only loaded initially, and will not change
+      after subsequent refreshes.
+    </i>
```

Comment:
Do we have a test for this + is it true for both course templates and traditional courses?

Maybe the traditional courses should remain functioning as they were?